lib/Mail.php

```diff
@@ -165,7 +174,9 @@ public function sendGuestPlainInviteMail($shareWith, $uid, $token) {
 
 		$this->logger->debug("sending invite to $shareWith: $registerLink", ['app' => 'guests']);
 
-		$subject = (string)$this->l10n->t('%s invited you', [$senderDisplayName]);
+		$useDefaultLanguage = $this->getDefaultLanguage();
```

Comment:
```suggestion
		$defaultLanguage = $this->getDefaultLanguage();
```
And adjust the var name in the code.